probe-rs-cli-util/src/argument_handling.rs

```diff
@@ -65,11 +62,11 @@ fn remove_arguments_test() {
     ];
 
     let mut arguments = vec![
+        "--chip-description-path=kek".to_string(),
+        "--chip-description-path".to_string(),
         "--chip=kek".to_string(),
         "--chip".to_string(),
         "kek".to_string(),
-        "--chip-description-path=kek".to_string(),
-        "--chip-description-path".to_string(),
         "kek".to_string(),
```

Comment:
This line is now duplicated.

Reply:
Hmm According to the full state it is not: https://github.com/probe-rs/probe-rs/blob/46e3454e6a7afea9f266e71f368811d64d41ff8a/probe-rs-cli-util/src/argument_handling.rs

Reply:
It is:

https://github.com/probe-rs/probe-rs/blob/46e3454e6a7afea9f266e71f368811d64d41ff8a/probe-rs-cli-util/src/argument_handling.rs#L69-L70

Reply:
Ok then its a bug in the webinterface again.